Parser/asdl_c.py

```diff
@@ -60,6 +60,15 @@ def reflow_lines(s, depth):
         lines.append(padding + cur)
     return lines
 
+def reflow_c_string(s, depth):
+    buffer = []
+    for index, line in enumerate(s.splitlines()):
+        if index == 0:
+            buffer.append('"%s"' % line)
+        else:
+            buffer.append('"\\n%s"' % line)
+    return ("\n%s" % (' ' * depth * TABSIZE)).join(buffer)
```

Comment:
It is more common to keep `"\n"` at the end of line, not at the begin.

And why not use `replace()`? The code can be much shorter.
```c
return '"%s"' % s.replace('\n', '\\n"\n%s"' % (' ' * depth * TABSIZE))
```